nnstreamer_example/custom_example_opencv/nnstreamer_customfilter_opencv_average.cc

```diff
@@ -0,0 +1,143 @@
+/**
+ * NNStreamer OpenCV Custom Filter Example: Average
+ * Copyright (C) 2018 Sangjung Woo <sangjung.woo@samsung.com>
+ *
+ * LICENSE: LGPL-2.1
+ *
+ * @file  nnstreamer_customfilter_opencv_average.cc
+ * @date  25 Oct 2018
+ * @brief  OpenCV Custom NNStreamer Filter Example: Average
+ * @author  Sangjung Woo <sangjung.woo@samsung.com>
+ * @bug  No known bugs
+ * @see  nnstreamer_customfilter_example_average.c
+ * 
+ * This example calculates the average value of input tensor for 
+ * each channel (i.e. R, G & B). The shape of the input tensor is
+ * [N][y][x][M] and that of the output tensor is [N][1][1][M]. 
+ */
+
+#include "opencv2/opencv.hpp"
+
+#include <glib.h>
+#include <tensor_filter_custom.h>
+#include <tensor_common.h>
+
+/**
+ * @brief _pt_data Internal data structure
+ */
+typedef struct _pt_data
+{
+  uint32_t in_height;  /***< height of input tensor */
+  uint32_t in_width;   /***< width of input tensor */
+  uint32_t in_channel; /***< channel of input tensor */
+} pt_data;
+
+/**
+ * @brief pt_init
```

Comment:
BTW, Could you describe @brief tag in more detail? the annotation is too short. :(
```bash
@brief   summarize a goal/meaning of the function

Describte in detail ASAP.............................................

.....................................................................

.....................................................................
```

Reply:
All nnstreamer-custom-filter has the same function lists and this is one of them. We'd better provide some "custom filter code generator" and add good doxygen statements in that code generator. So, it's ok not to have good doxygen brief entries for this common callback.